### PR TITLE
Sync with conda-forge

### PR DESCRIPTION
**Pull request by SC426**
sync with conda-forge

actual update will come in another PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+About tensorflow-metadata-feedstock
+===================================
+
+Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/tensorflow-metadata-feedstock/blob/main/LICENSE.txt)
+
+Home: https://pypi.org/project/tensorflow-metadata/
+
+Package license: Apache-2.0
+
+Summary: Library and standards for schema and statistics.
```
Comment (cbouss) [inline README.md:15]:
```suggestion
Updating tensorflow-metadata-feedstock
===================================

Although this isn't a fork of conda-forge's feedstock, it's a copy, with some changes to make it work for defaults. To update this feedstock to the latest version of the conda-forge one, pull conda-forge's updates using your favourite git mechanism, making sure that afterwards the changes from https://github.com/AnacondaRecipes/tensorflow-metadata-feedstock/pull/1 are still applied.

About tensorflow-metadata-feedstock
===================================

Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/tensorflow-metadata-feedstock/blob/main/LICENSE.txt)

Home: https://pypi.org/project/tensorflow-metadata/

Package license: Apache-2.0

Summary: Library and standards for schema and statistics.
```

**Review by cbouss — APPROVED** [review]
Suggested an explanation change on top of the readme. Otherwise, loogs good to me.

**Review by danpetry — APPROVED** [review]
you can do `git merge --strategy-option theirs` to automatically resolve merge conflicts by selecting theirs. That means you get what's on conda-forge's branch automatically

**Comment by skupr-anaconda** [comment]
After syncing with conda-forge the recipe should be reworked based on our recipe standards.

**Pull request merged**